### PR TITLE
SSNSPA-228 search filter UI polish

### PR DESCRIPTION
**Pull request by fabian202**

### CODE REVIEW

File: src/components/molecules/SearchPanel/index.js [hunk 1]
```diff
@@ -102,7 +109,11 @@ const SearchPanel = ({
   const handleChange = (e) => {
     setDateField(e);
 
-    const fil = { dateField: e.value };
+    if(!e) {
+      setDateFrom(null);
+      setDateTo(null);
+    }
```
Comment (fabian202) [inline src/components/molecules/SearchPanel/index.js:115]: @cesar82 take a look here!

File: src/components/molecules/SearchPanel/index.js [hunk 2]
```diff
@@ -252,7 +263,9 @@ const SearchPanel = ({
       case 'lastVerifiedOn':
         setDateFrom('');
         setDateTo('');
-        handleChange(filterDates[0]);
+        setDateField(null);
+        setDateFrom(null);
+        setDateTo(null);
```
Comment (fabian202) [inline src/components/molecules/SearchPanel/index.js:268]: @cesar82 also here

File: src/components/molecules/SearchPanel/index.js [hunk 3]
```diff
@@ -79,7 +86,7 @@ const SearchPanel = ({
       }
 
       //Ignore false empty or undefined properties
-      setFilters(defaultFilters.filter((item) => item && item !== 'all'));
+      setFilters(defaultFilters.filter((item) => item && item));
```
Comment (normandanielmq) [inline src/components/molecules/SearchPanel/index.js:89]: `.filter(item => item)`

File: src/components/molecules/SearchPanel/index.js [hunk 4]
```diff
@@ -874,9 +913,7 @@ const StyledInput = styled(Input)`
 `;
 
 const Title = ({ className, children }) => (
-  <div className={className}>
-    {children} <FaInfoCircle />
-  </div>
+  <div className={className}>{children}</div>
```
Comment (normandanielmq) [inline src/components/molecules/SearchPanel/index.js:916]: Do we need another component just for a `div`?

**Pull request merged**